addons/test_mail/tests/test_mail_message.py

```diff
@@ -388,11 +388,6 @@ def test_mail_message_access_search(self):
         messages = self.env['mail.message'].with_user(self.user_portal).search([('subject', 'like', '_ZTest')])
         self.assertFalse(messages)
 
-        # Test: Portal: 2 messages (public group with a subtype)
-        self.group_restricted_channel.write({'public': 'public'})
-        messages = self.env['mail.message'].with_user(self.user_portal).search([('subject', 'like', '_ZTest')])
-        self.assertEqual(messages, msg4 | msg5)
```

Comment:
this should probably be kept, create a public channel here to make the assert?

Reply:
I think the test case here is to see if the portal users see the messages when a group channel is changed to public channel. I think we can keep this test but make the group_public_id null rather than create a new public channel.